gremlin-core/pom.xml

```diff
@@ -61,6 +61,11 @@ limitations under the License.
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
yes - though - it will be removed by the time this pull request is over at this point